### PR TITLE
Make installation via homebrew the default instructions for macOS

### PR DESCRIPTION
**Pull request by nmccrory**
## The Problem/Issue/Bug:
Homebrew is the most straight forward and is the expected way of installing software for many macOS users.
## How this PR Solves The Problem:
This PR solves the problem by listing homebrew as the first installation instruction for macOS. Docs provide code examples to download, install, and upgrade ddev. As well as a link to the homebrew download/install instructions - that user's without homebrew may use as a reference.
## Related Issue Link(s):
#434

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,18 @@ ddev requires ports 80 and 3306 to be available for use on your system when site
 If you need to use another environment after using ddev, simply ensure all of your ddev sites are stopped or removed. ddev only uses system ports when at least one site is running.
 
 ## Installation
+### Homebrew - macOS
+
+For macOS users, we recommend downloading and installing ddev via homebrew:
```
Comment (rfay) [inline README.md]: Make homebrew a link here.

File: README.md [hunk 2]
```diff
@@ -32,6 +32,18 @@ ddev requires ports 80 and 3306 to be available for use on your system when site
...
+```
+brew tap drud/ddev && brew install ddev
+```
+To upgrade to the latest version of ddev, simply run:
```
Comment (rfay) [inline README.md]: Probably "Later, to upgrade to a newer version"

File: README.md [hunk 3]
```diff
@@ -32,6 +32,18 @@ ddev requires ports 80 and 3306 to be available for use on your system when site
...
+```
+brew upgrade ddev
+```
+If you need to install homebrew, instructions can be found [here](https://brew.sh/).
+  
 ### Installation Script - Linux and macOS
```
Comment (rfay) [inline README.md]: I'd just change this heading to "Installation Script - MacOS or Linux".

**Comment by nmccrory** [comment]
@rfay your changes have been made. Does this have the green light to squash and merge?

**Pull request merged**